### PR TITLE
Table name can be in multiple parts

### PR DESCRIPTION
**Pull request by pimeys**
In some cases the table name metadata comes in multiple parts. This caused corrupted data when reading the `IMAGE` type, and the PR should fix the bug in https://github.com/prisma/tiberius/issues/56.

### CODE REVIEW

File: src/tds/codec/token/token_col_metadata.rs [hunk 1]
```diff
@@ -168,40 +162,37 @@ impl BaseMetaDataColumn {
             TypeInfo::VarLenSized(VarLenType::Text, _, _) => {
                 src.read_u16_le().await?;
```
Comment (pimeys) [inline src/tds/codec/token/token_col_metadata.rs:163]: I really don't have any idea why `TEXT` has these 40 bytes of data before the parts count. There might be some information behind I don't know about and a bug that might come haunting later...

Comment (tomhoule) [reply src/tds/codec/token/token_col_metadata.rs:163]: yeah, they should be documented in the protocol specs, it's very weird if they aren't.

Comment (pimeys) [reply src/tds/codec/token/token_col_metadata.rs:163]: Yeah. It's probably somewhere. But my search with `TEXT` in the TDS manual PDF caused no results.

At least I found the table name parts from the manual finally:
> The fully qualified base table name for this column. It contains the table name length and table name. This exists only for text, ntext, and image columns. It specifies how many parts will be returned and then repeats PartName once for each NumParts.

Comment (wackydoo) [reply src/tds/codec/token/token_col_metadata.rs:163]: I confirmed that this branch returned the correct table names _and_ the correct blob data for the original query that I reported in #56 . I used the following code to check the blob data sha1sum was correct.

```
let first_result  = stream.into_row().await?.unwrap();
let blob = first_result.get::<&[u8],usize>(1).unwrap();
let mut m = sha1::Sha1::new();
m.update(blob);
println!("blob_sha1 = {}",m.digest().to_string());
```
Hopefully the second line is a reasonable idiomatic way to get a reference to the blob data.

Please merge.

**Pull request merged**